### PR TITLE
Upgrade Elixir to 1.5.2

### PR DESCRIPTION
**Pull request by michaelstalker**
[Card](https://www.pivotaltracker.com/story/show/521824492)

I'm not sure this PR is necessary. This library is a dependency for other apps we're upgrading to Elixir 1.5.2.

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -5,7 +5,7 @@ defmodule ExOps.Mixfile do
     [
       app: :ex_ops,
       version: "0.1.0",
-      elixir: "~> 1.4",
+      elixir: "~> 1.5",
```
Comment (michaelstalker) [inline mix.exs]: Do we want the change in line 8? It may cause issues if a host application is only set up to run Elixir 1.4.x.

Comment (alanvoss) [reply mix.exs]: Do you have an example?  I can't see why that would be an issue.  Force them to upgrade.

Comment (Ebtoulson) [reply mix.exs]: 👎 Upgrading might not be that simple for everyone. Remember this is a public repo. `~> 1.4` would cover 1.5

**Comment by michaelstalker** [comment]
I'm going to close this PR. ex_ops is compatible with Elixir 1.5, even if we don't require host apps to upgrade.

**Pull request closed without merging**